packages/reactive-element/src/legacy-decorators/custom-element.ts

```diff
@@ -58,8 +62,14 @@ const standardCustomElement = (
  * @param tagName The tag name of the custom element to define.
  */
 export const customElement =
-  (tagName: string) =>
-  (classOrDescriptor: CustomElementClass | ClassDescriptor) =>
-    typeof classOrDescriptor === 'function'
-      ? legacyCustomElement(tagName, classOrDescriptor)
-      : standardCustomElement(tagName, classOrDescriptor as ClassDescriptor);
+  (tagName: string): CustomElementDecorator =>
+  (
+    classOrTarget: CustomElementClass | unknown,
+    context?: ClassDecoratorContext<Constructor<HTMLElement>>
+  ) =>
+    (typeof classOrTarget === 'function'
```

Comment:
Later decorators this check swaps to `typeof classOrTarget === 'object'`, and then the first branch is the standard decorator, followed by the legacy decorator in the `else`.

Is one check preferred over the other. No action, just curious.

Reply:
There's no real difference.